### PR TITLE
Added sqrt to the goodie caclulator

### PR DESCRIPTION
**Pull request by jgkamat**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Added sqrt functionality to the DDG caclulator

**What problem does your Instant Answer solve (Why is it better than organic links)?**
People who want to use DDG to find the square root of something usually type out 'sqrt(x)' instead of 'x^0.5'

**What is the data source for your Instant Answer? (Provide a link if possible)**
Math?

**Why did you choose this data source?**
N/A

**Are there any other alternative (better) data sources?**
N/A

**What are some example queries that trigger this Instant Answer?**
sqrt(4), sqrt(pi), sqrt(4+23)

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Math geeks, students

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No

**Which existing Instant Answers will this one supercede/overlap with?**
Calculator!

**Are you having any problems? Do you need our help with anything?**
I wanted the input query 'sqrt 4' to trigger as well with no parentheses (along with 'sin 4'), but I don't want to mess with the calculator too much before I fully understand how it works. Maybe another time...

**Where did you hear about DuckDuckHack? (For first time contributors)**
Gatech Linux User Group

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
snapshot78

## Checklist
Please place an 'X' where appropriate.

```
- [N/A] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [x] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Linux 3.16
        - [x] Google Chrome
        - [x] Firefox
        - [x] Opera

    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: t/Calculator.t [hunk 1]
```diff
@@ -452,6 +452,15 @@ ddg_goodie_test(
...
+    'sqrt(4)' => test_zci(
+        'sqrt(4) = 2',
+        heading           => 'Calculator',
+        structured_answer => {
+            input     => ['sqrt(4)'],
+            operation => 'Calculate',
+            result    => qr/>2</
+        }
+    ),
     '1.0 + 5 squared' => test_zci(
```
Comment (mintsoft) [inline t/Calculator.t]: Can you add some tests of this in a broader context?

**Comment by mintsoft** [comment]
Thanks for the contribution @jgkamat it looks like it'll just work, however I think some slightly more covering tests would be advantageous. I'll also give this a manual run-around, given the importance of the Calculator IA.

**Comment by jgkamat** [comment]
sure thing! I'll add some more tests of the sqrt function in a bit!

**Comment by mintsoft** [comment]
This LGTM, but can someone else (@moollaza @jagtalon @MrChrisW @killerfish) give this a quick run-around for something I may have overlooked; given the important of the Calculator IA?

**Comment by MrChrisW** [comment]
@mintsoft LGTM :+1: I've tested this and see no issues.

**Comment by mintsoft** [comment]
Cool thanks @MrChrisW

**Comment by mintsoft** [comment]
Thanks for the enhancement @jgkamat !

**Comment by moollaza** [comment]
:+1:

**Comment by jgkamat** [comment]
Thanks so much for your time! =D

**Pull request merged**